src/signalcorr.jl

```diff
@@ -89,10 +103,17 @@ autocov{T<:Real}(x::VecOrMat{T}; demean::Bool=true) = autocov(x, default_autolag
 
 ## autocor
 
+"""
+    autocor!(r, x, lags; demean=true)
+
+Compute the autocorrelation function (ACF) of a vector or matrix `x` at `lags`
```

Comment:
Is "function" actually correct? Just double-checking, I'm not familiar with this method.